### DISCUSSION TITLE
Change string references to use nameof(...)

### DISCUSSION THREAD

**Discussion by Montago** [codecadwallader/codemaid#988] [Ideas]
eg: 

In my codebase i have a ton of OnPropertyChanged("Property") which should be changed to OnPropertyChanged(nameof(Property))
The same applies to DependencyProperty.Register("PropertyName".....)

The conversion should be intelligent, by checking if the property actually exists to not cause bugs.

**Comment by codecadwallader** [comment]
Thanks for that suggestion.  That deeper analysis of the references for refactoring is probably a bit out of scope of what CodeMaid currently handles.  It sounds like this may be available as a code fix according to https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1507
